packages/python-packages/apiview-gpt/src/__init__.py

```diff
@@ -13,7 +15,7 @@ def console_entry_point():
     print("Running apiview-gpt version {}".format(__version__))
     reviewer = GptReviewer()
     # FIXME: Make this a proper CLI
-    input_filename = "test2.txt"
+    input_filename = "test.txt"
```

Comment:
It looks like this file got deleted?

Reply:
Well, now you actually pass in the value, so I should refactor this logic away.